gateway/src/main/scala/filodb/gateway/GatewayServer.scala

```diff
@@ -151,7 +155,10 @@ object GatewayServer extends StrictLogging {
                                                   otelCumulativeHistogram)
                    else if (genOtelDeltaHistData) TestTimeseriesProducer.genHistogramData(startTime, numSeries,
                                                   otelDeltaHistogram)
-                   else if (genDeltaHist) TestTimeseriesProducer.genHistogramData(startTime, numSeries, deltaHistogram)
+                   else if (genOtelExpDeltaHistData) TestTimeseriesProducer.genHistogramData(startTime, numSeries,
+                     otelDeltaHistogram, otelExponential = true)
```

Comment:
IMPORTANT: I think you are planning to use the existing `otelDeltaHistogram` schema, which makes sense. 

In that case, we would also need to update `InputRecord.writeOtelDeltaHistRecord` (currently it is hard-coded to use Custom-Buckets, we would need to make a change in this function such that the gateway can specify the bucket scheme and we create the buckets accordingly ).

Reply:
Yup. I made that change and will send out another commit for this PR.